Lesson_16.py

```diff
@@ -0,0 +1,52 @@
+class Manager:
```

Comment:
Please decrease number of files - here should be only 3 files ( TeamLead, Shape and file with tests)